node/node_handler.go

```diff
@@ -57,13 +57,6 @@ func (node *Node) processSkippedMsgTypeByteValue(
 
 // HandleMessage parses the message and dispatch the actions.
 func (node *Node) HandleMessage(content []byte, sender libp2p_peer.ID) {
-	// log in-coming metrics
-	node.host.LogRecvMessage(content)
```

Comment:
why this is removed? do we have better metrics?

Reply:
It was done wrong , the metrics is incredibly noisy in log , and more than that , it is not hooked up correctly to the host , just stand alone at moment. After have stability on PR, will hook it back on correctly